### PR TITLE
【修改】修改imx6ull-smart中mnt.c的逻辑错误

### PR DESCRIPTION
**Pull request by zhuzhuzhus**
## 拉取/合并请求描述：(PR description)

[

#### 为什么提交这份PR (why to submit this PR)
ART-PI如果mountemmc失败的话会直接退出，不会继续mountsd卡

#### 你的解决方案是什么 (what is your solution)
修改mnt.c

#### 请提供验证的bsp和config (provide the config and bsp) 

- BSP:

- .config:

- action:

]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 代码是高质量的 Code in this PR is of high quality
- [x] 已经使用[formatting](https://github.com/mysterywolf/formatting) 等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

File: bsp/nxp/imx/imx6ull-smart/applications/mnt.c [hunk 1]
```diff
@@ -20,25 +20,23 @@ int mnt_init(void)
...
-    else
-    {
-        rt_kprintf("emmc file system initialization done!\n");
-    }
-
     if (dfs_mount("sd0", "/", "elm", 0, (void *)part_id) != 0)
     {
-        rt_kprintf("Dir / mount failed!\n");
-        return -1;
+         if (dfs_mount("emmc0", "/", "elm", 0, (void *)part_id) != 0)
```
Comment (polarvid) [inline bsp/nxp/imx/imx6ull-smart/applications/mnt.c:25]: 请调整一下代码对齐

File: bsp/nxp/imx/imx6ull-smart/applications/mnt.c [hunk 2]
```diff
@@ -20,25 +20,23 @@ int mnt_init(void)
...
-    else
-    {
-        rt_kprintf("emmc file system initialization done!\n");
-    }
-
     if (dfs_mount("sd0", "/", "elm", 0, (void *)part_id) != 0)
     {
-        rt_kprintf("Dir / mount failed!\n");
-        return -1;
+        if (dfs_mount("emmc0", "/", "elm", 0, (void *)part_id) != 0)
```
Comment (BernardXiong) [inline bsp/nxp/imx/imx6ull-smart/applications/mnt.c:25]: 可以默认装载emmc，然后再尝试装载sd卡吧

Comment (zhuzhuzhus) [reply bsp/nxp/imx/imx6ull-smart/applications/mnt.c]: ok,已更改

**Review by BernardXiong — APPROVED** [review]

**Comment by Rbb666** [comment]
mnt.c line[39]: please delete extra space at the end of this line.：请在最后一行结尾加上换行

File: bsp/nxp/imx/imx6ull-smart/applications/mnt.c [hunk 3]
```diff
@@ -35,8 +35,7 @@ int mnt_init(void)
     else
     {
         rt_kprintf("emmc file system initialization done!\n");
-    }
-    
+    }
```
Comment (polarvid) [inline bsp/nxp/imx/imx6ull-smart/applications/mnt.c:38]: 这里行尾有空格呢。提交之前可以自己用工具扫一下。

**Review by polarvid — CHANGES_REQUESTED** [review]

**Review by polarvid — APPROVED** [review]

**Review by BernardXiong — APPROVED** [review]

**Pull request merged**